internal/commands/config.go

```diff
@@ -20,6 +20,7 @@ func NewConfigCommand(logger logging.Logger, cfg config.Config, cfgPath string,
 	cmd.AddCommand(ConfigExperimental(logger, cfg, cfgPath))
 	cmd.AddCommand(ConfigTrustedBuilder(logger, cfg, cfgPath))
 	cmd.AddCommand(ConfigRunImagesMirrors(logger, cfg, cfgPath))
+	cmd.AddCommand(ConfigPullPolicy(logger, cfg, cfgPath))
```

Comment:
I wasn't sure if this new command should be under "experimental" or not. If it is still considered "experimental," I can move it there.